MekHQ/src/mekhq/gui/CampaignGUI.java

```diff
@@ -2078,6 +2052,8 @@ protected void loadPlanetTSVFile() {
      * @param filename
      */
     protected void exportPlanets(FileType format, String dialogTitle, String filename) {
+        //TODO: Fix this
```

Comment:
Did we decide if we're keeping this or not?

Reply:
For this PR, we are leaving it out. It was apparently for the SUCS project but @NickAragua says they are using a different format now anyway.